src/styles.css

```
@@ -59,4 +62,4 @@ body {
 
 .btn-primary:active {
   background-color: #212020;
-}
+}
```

Append new-line to EOF
```suggestion
}

```